frontend/src/components/course/components/modules-cards-container/components/module-card/styles.module.scss

```diff
@@ -54,3 +55,7 @@
   align-items: center;
   padding-right: 10px;
 }
+
+.statusContainer {
+  width: 115px;
```

Comment:
What if status will be more than this width?

Reply:
we don't have statuses more than this width
the longest one is 'uncompleted', and it fits
I suppose if we have a longer one in the future, this can be easily updated